news/_posts/releases/2022-01-26-prestashop-1-7-8-3-maintenance-release.md

```diff
@@ -0,0 +1,84 @@
+- --
+layout: post
+title:  "Release Of PrestaShop 1.7.8.3"
+subtitle: "MAINTENANCE VERSION OF THE 1.7.8.X BRANCH"
+date:   2022-01-24 14:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2021/12/1.7.8.3_banner.jpg
+icon: icon-checked
+tags:
+- version
+- patch
+- releases
+
+---
+
+PrestaShop 1.7.8.3 is now available. This maintenance release fixes 25 bugs and 1 secrurity issue since the 1.7.8.2 release.
+Note : 2 of this fixes break the backward compatibility ( ([#26933][https://github.com/PrestaShop/PrestaShop/issues/26933](https://github.com/PrestaShop/PrestaShop/issues/26933)) and ([#26643][https://github.com/PrestaShop/PrestaShop/issues/26643](https://github.com/PrestaShop/PrestaShop/issues/26643)) ) 
+
+*![1.7.8.3 is available!](/assets/images/2022/01/1.7.8.3_banner.jpg)*
+
+
+We suggest upgrading your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before.
+
+Reminder: the 1-Click Upgrade module’s latest version is v4.13.0, don’t forget to upgrade it.
```

Comment:
Now it's v4.14.0

https://github.com/PrestaShop/autoupgrade/releases/tag/v4.14.0

Reply:
I think v4.14.0 was released in order to make sure bug does not happen if users upgrade to 1.7.8.3

https://github.com/PrestaShop/autoupgrade/pull/448

So we might add a sentence about "please v4.14.0, don't use v4.13.0"

Reply:
+1